packaging/standalone/src/main/distribution/shell-scripts/bin/neo4j-import

```diff
@@ -31,6 +31,11 @@ build_classpath
 
 EXTRA_JVM_ARGUMENTS="-Dfile.encoding=UTF-8"
 
+cat >&2 <<EOF
+WARNING: neo4j-import is deprecated and support for it will be removed in a future
+         version of Neo4j; please use neo4j-admin import --mode=csv instead.
```

Comment:
I don't think we should mention `--mode=csv`. The user can do `--help` to figure stuff out. (otherwise this message will probably go out of date real quick)